src/main/java/uk/nhs/hee/tis/trainee/forms/config/Config.java

```diff
@@ -34,6 +35,8 @@
 @Configuration
 public class Config {
 
+  public static final String CUSTOM_STRING_DESERIALIZER_MODULE_NAME = "customStringDeserializer";
```

Comment:
I agree giving the module a specific name seems unnecessary; the issue I had is that the test to see whether it's registered correctly wont work as it does for the JavaTimeModule. It gives 'SimpleModule-1' in getRegisteredModuleIds() if its not assigned a specific name - and 'SimpleModule-2' when creating a new one to compare it to, so the comparison fails 🤔